modules/using-a-drift-metric-in-a-credit-card-scenario.adoc

```diff
@@ -0,0 +1,207 @@
+
+Mean shift compares a numeric test dataset against a numeric training dataset. It produces a p-value that measures the probability the test data has originated from the same numeric distribution as the training data. A p-value less than 0.05 indicates a statistically significant drift between the two datasets. A p-value equal to or greater than 0.05 indicates no statistically significant evidence of drift.
+
+[NOTE]
+====
+Mean shift performs best when each feature in the data is normally distributed. Choose a different metric for working with different or unknown data distributions.
+====
+
+.Prerequisites
+
+ifdef::upstream[]
+* Your cluster administrator added you as a user to the cluster and link:{odhdocshome}/monitoring-data-science-models/#configuring-trustyai_monitor[configured TrustyAI] for use in the project that contains the deployed models.
+
+* You set up TrustyAI for your project, as described in link:{odhdocshome}/monitoring-data-science-models/#setting-up-trustyai-for-your-project_monitor[Setting up TrustyAI for your project].
+endif::[]
+//downstream
+ifndef::upstream[]
+* Your cluster administrator added you as a user to the cluster and link:{rhoaidocshome}{default-format-url}/monitoring_data_science_models/configuring-trustyai_monitor[configured TrustyAI^] for use in the project that contains the deployed models.
+
+* You set up TrustyAI for your project, as described in link:{rhoaidocshome}{default-format-url}/monitoring_data_science_models/setting-up-trustyai-for-your-project_monitor[Setting up TrustyAI for your project^].
+endif::[]
+
+
+.Procedure
+
+. Obtain a bearer token to authenticate your external endpoints by running the following command:
++
+[source, bash]
+----
+$ oc apply -f resources/service_account.yaml
+export TOKEN=$(oc create token user-one)
+----
+
+. In your model namespace, deploy the storage container, serving runtime, and the credit model: 
++
+[source, bash]
+----
+$ oc project model-namespace || true
+$ oc apply -f resources/model_storage_container.yaml
+$ oc apply -f resources/odh-mlserver-1.x.yaml
+$ oc apply -f resources/model_gaussian_credit.yaml
+----
+
+. Set the route for your data upload:
++
+[source, bash]
+----
+TRUSTY_ROUTE=https://$(oc get route/trustyai-service --template={{.spec.host}})
+----
+
+. Download the training data payload (file size 472 KB):
++
+[source, bash]
+----
+wget https://github.com/trustyai-explainability/odh-trustyai-demos/blob/72f748da9410f92a60bea73ce5e3f47c10ad1cea/3-DataDrift/kserve-demo/data/training_data.json -O training_data.json 
+----
+
+. Label your model training data. This data has four main fields. The `model_name` and `data_tag` fields require a label because they are directly referenced in the Metrics dashboard later in the scenario. In addition to the required fields, it is recommended to also label response and request fields. The four fields are:
```

Comment:
```suggestion
. Label your model training data. This data has four main fields. The `model_name` and `data_tag` fields require a label because they are directly referenced in the Metrics dashboard later in the scenario. In addition to the required fields, it is best to also label response and request fields. The four fields are:
```